pkg/apis/config/defaults.go

```diff
@@ -103,12 +119,24 @@ func (d *Defaults) GetBrokerConfig(ns string) (*BrokerConfig, error) {
 		return nil, errors.New("Defaults are nil")
 	}
 	value, present := d.NamespaceDefaultsConfig[ns]
+	// get namespace default config
 	if present && value.BrokerConfig != nil {
 		return value.BrokerConfig, nil
 	}
```

Comment:
and that makes me think we need a test for this case as well.

Reply:
> namespace

I see your point, I think the broker-class based default at the namespace level we may not use it , but once we set the the single value.BrokerConfig we surely want to use it. And is it necessary to set broker-class based default at the namespace level ? Because we already set it at the cluster level, what do you think?

Reply:
> If I specify broker-class based default at the namespace level `value.BrokerClasses` I don't think this would work because we returning always the single `value.BrokerConfig`

I see your point, I think the broker-class based default at the namespace level we may not use it , but once we set the the single value.BrokerConfig we surely want to use it. And is it necessary to set broker-class based default at the namespace level ? Because we already set it at the cluster level, what do you think?